packages/fxa-auth-db-mysql/scripts/read-security-events.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node readSecurityEvents.js uid
+
+const log = {
+  trace: console.log,
+  error: console.error,
+  stat: console.log,
+  info: console.log
+} //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+
+// This input of user's data can be modified according to the usecase
+const user = {
+  // email: 'it may also be used for getting security info',
+  uid: process.argv[2]
+}
+
+function readEvents() {
+  DB.connect(config).then(async db => {
+    try {
+      const devices = await readDevices(db)
+      securityEventsObject.devices = devices
+
+      const sessions = await readSessionTokens(db)
+      securityEventsObject.sessions = sessions
+
+      const events = await readSecurityEvents(db)
+      securityEventsObject.securityEvents = events
+
+      log.info(securityEventsObject)
+
+      db.close()
+    } catch (err) {
+      log.error(err.stack || err.message || err)
+      process.exit()
+    }
+  })
+}
+
+async function readDevices(db) {
+  try {
+    let { uid } = user
+    uid = Buffer.from(uid, 'hex')
```

Comment:
`Buffer.from(uid, 'hex')` is repeated 3 times, maybe just set user.uid to that version everywhere.